pom.xml

```diff
@@ -1,194 +1,149 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>eu.bitwalker</groupId>
-	<name>User-Agent-Utils</name>
-	<url>http://www.bitwalker.eu/software/user-agent-utils</url>
-	<artifactId>UserAgentUtils</artifactId>
-	<version>1.16-SNAPSHOT</version>
-	<description>Utility classes to handle user-agents.</description>
-	<properties>
-		<github.global.server>github</github.global.server>
-	</properties>
-	<licenses>
-		<license>
-			<name>New BSD License</name>
-			<url>
-				http://user-agent-utils.googlecode.com/svn/trunk/UserAgentUtils/LICENSE.txt
-			</url>
-			<distribution>repo</distribution>
-		</license>
-	</licenses>
-	<developers>
-		<developer>
-			<id>haraldwalker</id>
-			<name>Harald Walker</name>
-			<url>http://www.bitwalker.eu</url>
-			<roles>
-				<role>developer</role>
-			</roles>
-			<timezone>-2</timezone>
-		</developer>
-	</developers>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>eu.bitwalker</groupId>
+    <name>User-Agent-Utils</name>
+    <url>http://www.bitwalker.eu/software/user-agent-utils</url>
+    <artifactId>UserAgentUtils</artifactId>
+    <version>1.16.${build.number}-onshape</version>
```

Comment:
I would put onshape in the artifact id not the version. This will guarantee there will be no ambiguity about which artifact we are referring to.